src/api-v1.js

```diff
@@ -90,10 +91,12 @@ api.declare({
     let result;
 
     do {
+      let startTime = new Date();
```

Comment:
This isnt execution time... That said he would wrap the `do {... } while(..)` loop inside a:
```js
this.monitor.timedHandler('cache-lookup-duration', async () => {
  do {....} while(...);
});
```

-------
Again, off topic here...

@imbstack, after seeing the pattern above... I'm tempted to suggest we rename `timedHandler` to `time` or `timer`, and that we return the value of the `handler`, like:
```js
let val = await this.monitor.time('cache-lookup-duration', async () => {
  return 24
});
assert(val == 24);
```

This way, we could use it like:
```js
let status = await this.monitor.time('task-status', queue.status(taskId));
```
So `monitor.time(prefix, handlerOrPromise)` either gets a function, an async function or a promise that it must measure the time of... and must return the same promise...